package.json

```diff
@@ -36,10 +36,11 @@
     "connected-react-router": "~6.9.0",
     "history": "~4.10.0",
     "immutable": "4.0.0-rc.10",
-    "lattice-ui-kit": "~0.46.0",
+    "lattice-ui-kit": "~0.46.1",
     "lodash": "~4.17.21",
     "loglevel": "~1.7.0",
     "polished": "~4.1.2",
+    "punycode": "^2.1.1",
```

Comment:
What is this for and where is it used?

Reply:
i literally have no idea how or why that's there... i removed it.